src/transaction.js

```diff
@@ -7,35 +7,20 @@ const typeforce = require('typeforce')
 const types = require('./types')
 const varuint = require('varuint-bitcoin')
 
-function varSliceSize (someScript) {
+const varSliceSize = (someScript) => {
   const length = someScript.length
```

Comment:
why?

Reply:
Personal preference. hoisted functions make me feel dirty inside nowadays... I'm not religious about it tho.

Reply:
for now, lets leave this as it was,  and then we can discuss the new style separately?